src/components/widget/MultiSelect.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class MultiSelect extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: this.props.options,
+      filter: '',
+      checkedItems: {},
+    };
+  }
+
+  componentDidMount() {
+    this.props.onFocus();
+  }
+
+  static getDerivedStateFromProps(nextProps) {
+    let selected =
+      nextProps.selectedItems && Array.isArray(nextProps.selectedItems)
```

Comment:
can we not have nested ternaries plx ?

Reply:
Yeah that I missed. Will fix asap. Thanks

Reply:
Fixed.